controllers/stats/index.js

```diff
@@ -0,0 +1,23 @@
+import db from '../../db/models';
+
+export default {
+  update: async (req, res) => {
+    try {
+      const { params: { slug } } = req;
+      const article = await db.Article.findOne({
+        where: { slug }
+      });
+
```

Comment:
Great work done. However, please check if an article exists before going ahead to increment the article reads.

Reply:
thanks for bringing this to my attention. will do that now